src/pho.php

```diff
@@ -187,6 +187,21 @@ function expandPaths($paths)
     require_once($path);
 }
 
+// Bootstrap file must be required directly rather than from function
+// invocation to preserve any loaded globals
+if ($console->options['bootstrap']) {
```

Comment:
Mind adding `$bootstrap = $console->options['bootstrap'];` above to shorten some of these lines a bit?